tests/io_perfomance/image-volume/run-iostat.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+#Running IOSTAT
+for (( i = 1; i <= 36; i++ ))
```

Comment:
how is this 1 - 36 determined? I thought you wanted to run run iostat for as long as fio is running, no?